gluoncv/model_zoo/rcnn/rcnn.py

```diff
@@ -151,6 +151,25 @@ def set_nms(self, nms_thresh=0.3, nms_topk=400, post_nms=100):
         self.nms_topk = nms_topk
         self.post_nms = post_nms
 
+    def reset_class(self, classes):
+        """Reset class categories and class predictors.
+
+        Parameters
+        ----------
+        classes : iterable of str
+            The new categories. ['apple', 'orange'] for example.
+
+        """
+        self._clear_cached_op()
+        self.classes = classes
+        self.num_class = len(classes)
+        with self.name_scope():
+            self.class_predictor = nn.Dense(
+                self.num_class + 1, weight_initializer=mx.init.Normal(0.01))
```

Comment:
use `prefix=self.class_predictor.prefix` to restore original name could be better